static/options.interfaces.ts

```diff
@@ -23,23 +23,33 @@
 // POSSIBILITY OF SUCH DAMAGE.
 
 export interface LibraryVersion {
-    version: string;
+    alias: string[];
+    dependencies: string[];
     hidden: boolean;
+    libId: string;
+    liblink: string[];
```

Comment:
Should we really be exposing these properties, as the client shouldn't really be doing anything with them.

Reply:
libId is used by the compiler pane, but you're right, liblink could be safely removed

Reply:
To be complete, these could be removed:
```
    libpath: string[];
    options: string[];
    path: string[];
    staticliblink: string[];
```